client/dashboard/task-list/index.js

```diff
@@ -409,13 +413,19 @@ class TaskDashboard extends Component {
 }
 
 export default compose(
+	window.wcSettings.preloadOptions
+		? withOptionsHydration( {
+				...window.wcSettings.preloadOptions,
+		  } )
+		: () => ( {} ),
```

Comment:
This should be `identity` from lodash: https://github.com/woocommerce/woocommerce-admin/pull/4094/commits/a22d7493210b90117bfbd1230164913ace6dbfe1#diff-713853358fa08698c1bf0f8dfa42ca74R282

Reply:
Done, thanks!

Reply:
I also moved this section to the bottom of the `compose` pile.